packages/taiko-client/proposer/transaction_builder/calldata.go

```diff
@@ -69,31 +76,63 @@ func (b *CalldataTransactionBuilder) Build(
 	signature[64] = uint8(uint(signature[64])) + 27
 
 	var (
-		to   = &b.taikoL1Address
-		data []byte
+		to            = &b.taikoL1Address
+		data          []byte
+		encodedParams []byte
```

Comment:
ditto

Reply:
don't think we are decoding anything here, no?